src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Media/Converters/Generated/MatrixValueSerializer.cs

```diff
@@ -9,7 +9,11 @@
 // Please see MilCodeGen.html for more information.
 //
 
+using MS.Internal;
```

Comment:
As you mentioned earlier, these extra imports are needed because one of the files being generated by the generator needs these imports ?

Also, I think there can be multiple generators that are involved while generating a file ?

Reply:
One generator can generate multiple files and each files can require different usings. The generator needs to include the usings by every files, otherwise it would need to dynamically add usings according to what code gets generated. Technically it could be done, I haven't done it because I didn't see the value in doing it for my inital PRs and I'm not sure making the generators more complicated just to trim the list of usings is a positive.